services/gitlab/gitlab-pipeline-status.service.js

```diff
@@ -56,10 +56,8 @@ module.exports = class GitlabPipelineStatus extends BaseSvgScrapingService {
       {
         title: 'Gitlab pipeline status (self-hosted)',
         pattern: ':user/:repo',
-        namedParams: { user: 'gitlab-org', repo: 'gitlab-ce' },
-        // It would be good to find an example of a publicly accessible, self-
-        // hosted gitlab.
-        query: { gitlab_url: 'https://gitlab.com' },
+        namedParams: { user: 'GNOME', repo: 'pango' },
+        query: { gitlab_url: 'https://gitlab.gnome.org' },
```

Comment:
It might not be directly related to the changes in this PR (I suspect it was introduced in #2308 ), but I noticed that the query gets passed through to the placeholder example but not the one with the params filled in:

![screenshot at 2018-11-18 12-46-46](https://user-images.githubusercontent.com/6025893/48672626-3c741c00-eb30-11e8-951b-2bba879b231a.png)

Reply:
Indeed. I noticed that too: #2339

Reply:
one step ahead :athletic_shoe:

Reply:
Since this is approved why don't we merge this. It'll make #2339 easier to review. I probably should have just added that change here.